bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -202,7 +198,9 @@ public BigtableBufferedMutator(
         heapSizeManager.getMaxInFlightRpcs());
     this.options = options;
     this.heapSizeManager = heapSizeManager;
-    this.executorService = executorService;
+    this.executorService =
```

Comment:
This is a bit weird, this depends on "executorService" & "AsyncMutatorCount".
Why don't we always depend on AsyncMutatorCount. It it simpler:

public BigtableBufferedMutator(...)
  this(... options.getAsyncMutatorCount() > 0 ? createDefaultExecutorService() : null);
}

@VisibleForTesting
public BigtableBufferedMutator(...)
  ...
  this.executorService = executorService;
}